modules/appeals_api/spec/factories/evidence_submissions.rb

```diff
@@ -4,15 +4,18 @@
   factory :evidence_submission, class: 'AppealsApi::EvidenceSubmission' do
     sequence(:id) { |n| n }
     guid { SecureRandom.uuid }
-    association :supportable, factory: :notice_of_disagreement
     upload_submission { create(:upload_submission, guid: SecureRandom.uuid) } # set the guid to pass uniqueness check
 
     trait :with_detail do
       detail { SecureRandom.alphanumeric(150) }
     end
 
-    trait :with_nod do
+    trait :for_nod do
       supportable { create(:notice_of_disagreement) }
     end
+
```

Comment:
Sorry I should've been a little clearer - is there a case where we're setting up the supportable and not creating a record? i.e - can we combine these two traits?